samples/charger/bq24190/boards/nucleo_f401re.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <zephyr/dt-bindings/i2c/i2c.h>
+
+&arduino_i2c {
+    status = "okay";
+    clock-frequency = <I2C_BITRATE_STANDARD>;
+
+    bq24190: bq24190@6b {
+        compatible = "ti,bq24190";
+        reg = <0x6b>;
+        status = "okay";
+
+        constant-charge-current-max-microamp = <1000000>;
+        constant-charge-voltage-max-microvolt = <4208000>;
+    };
```

Comment:
If this is a shield then it should be in a shield definition, if it's just a custom setup you have for testing it should not be upstreamed. Instead there should be `build_all` test to test build all the drivers and that's it.

Reply:
do you have an example?

Reply:
Oh sorry, I meant an example of the tests, but I found plenty after reading your https://github.com/zephyrproject-rtos/zephyr/pull/62551#discussion_r1345989207. 

This is just a custom fly-wired setup from the eval board to a Nucelo F401RE eval board, so it sounds like it shouldn't be here.

Reply:
Oh sorry the build_all bit, yeah load of them in `tests/drivers`, arguably every driver category should be there.